caper/caper_backend.py

```diff
@@ -230,6 +237,11 @@ def __init__(self, gcp_prj, out_gcs_bucket, concurrent_job_limit=None):
 
         if concurrent_job_limit is not None:
             config['concurrent-job-limit'] = concurrent_job_limit
+        if call_caching_dup_strat is not None:
+            assert call_caching_dup_strat in (
+                CaperBackendGCP.CALL_CACHING_DUP_STRAT_REFERENCE,
+                CaperBackendGCP.CALL_CACHING_DUP_STRAT_COPY)
```

Comment:
Error message in case the assertion fails would be good I think.

Reply:
Thanks, as I mentioned above, this will also be fixed in the latest PR.